ndcube/ndcube.py

```diff
@@ -792,3 +801,90 @@ def plot(self, *args, **kwargs):
                 "no default plotting functionality is available.")
 
         return self.plotter.plot(*args, **kwargs)
+
+    def _new_instance_from_op(self, new_data, new_unit):
```

Comment:
This also needs to carry over `global_coords`

Reply:
I will add that in. Is it a kwarg or does it need to be attached after instantiating like `extra_coords`?

Reply:
It needs to be attached.  I think the way to do it is attach the hidden version of global_coords to the new object.
`new_cube._global_coords = self._global_coords`